### DISCUSSION TITLE
How to replicate demo layer manager in react js?

### DISCUSSION THREAD

**Discussion by besart-k** [GrapesJS/grapesjs#5275] [Q&A]
Hello, 
I was trying to replicate demo layout manager but I couldn't fully manage to replicate, I had problems like freezing element when moving or showing indicator on canvas in which parent component is going to drop.

Is there somewhere a react demo, or maybe vue js but exactly like in grapesjs demo.

**Comment by artf** [comment]
Here you can find the custom layers example (in Vue): https://grapesjs.com/docs/modules/Layers.html#customization
Check also [this](https://github.com/GrapesJS/grapesjs/issues/4497#issuecomment-1213078160) which might be related to the freeze in Vue

**Reply by besart-k** [reply]
Thanks for the reply, what about the indicators on the parent/children in canvas and in layer manager also?

**Reply by artf** [reply]
With a custom layer manager, you have full control over the UI so you can handle indicators as you wish.
Regarding the canvas, at the moment, you can only change the style by targeting those elements with CSS.
